addons/event_sale/tests/test_event_sale.py

```diff
@@ -376,7 +384,200 @@ def test_ticket_price_with_pricelist_and_tax(self):
+                                   delta=precision_delta * qty)
+
+        # Only vary price list currency because product and event currency are related to company.currency (see models)
+        for currency in (eur_currency, usd_currency):
+            for is_product_tpl in (False, True):
+                descr = f'(using {"product template" if is_product_tpl else "product"}, price list in {currency.name})'
+                ticket, sale_order, sale_order_line = create_sale_order(
+                    is_product_templ=is_product_tpl,
+                    product_lst_price=30.0, ticket_price=100.0, qty=1,
+                    rule_type='formula', rule_amount=20.0, min_qty=2,
+                    price_list_currency=currency
+                )
+                assert_amount_total(sale_order, ticket,
+                                    convert_to_sale_order_currency(110),
+                                    qty=1,
+                                    msg=f"Ticket is $100 + 10% tax -> 110 {descr}")
+                sale_order_line.write({'product_uom_qty': 2})
+                assert_amount_total(sale_order, ticket,
+                                    convert_to_sale_order_currency(176),
+                                    qty=2,
+                                    msg=f"2 tickets at $100 -20%  -> $80 -> $160 + 10% tax -> 176 {descr}")
+
+                ticket, sale_order, sale_order_line = create_sale_order(
+                    is_product_templ=is_product_tpl,
+                    product_lst_price=30.0, ticket_price=100.0, qty=1,
+                    rule_type='formula', rule_amount=20.0, min_qty=1,
+                    price_list_currency=currency
+                )
+                assert_amount_total(sale_order, ticket,
+                                    convert_to_sale_order_currency(88),
+                                    msg=f"Ticket at $100 -20%  -> $80 + 10% tax -> 88 {descr}",
+                                    qty=1)
+
+                ticket, sale_order, sale_order_line = create_sale_order(
+                    is_product_templ=is_product_tpl,
+                    product_lst_price=30.0, ticket_price=100.0, qty=1,
+                    rule_type='fixed', rule_amount=10.0, min_qty=2,
+                    price_list_currency=currency
+                )
+                assert_amount_total(sale_order, ticket,
+                                    convert_to_sale_order_currency(110),
+                                    msg=f"Ticket is $100 + 10% tax -> 110 {descr}",
+                                    qty=1)
+                sale_order_line.write({'product_uom_qty': 2})
+                assert_amount_total(sale_order, ticket,
+                                    22,  # No conversion because the price is expressed in price list currency
+                                    msg=f"2 tickets at $10 +10% -> 22 {descr}",
+                                    qty=2)
```

Comment:
So sometimes its EUR right?
```suggestion
                ten_currency = '$10' if currency == usd_currency else '10€'
                assert_amount_total(sale_order, ticket,
                                    22,  # No conversion because the price is expressed in price list currency
                                    msg=f"2 tickets at {ten_currency} +10% -> 22 {descr}",
                                    qty=2)
```

Reply:
Changed